### PR TITLE
EOS-17582:[s3server Mini-provisioner] Pre-Requisite 3rd party package python36-ldap

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: root <root@ssc-vm-2452.colo.seagate.com>

### CODE REVIEW

File: srv/components/misc_pkgs/openldap/install.sls [hunk 1]
```diff
@@ -27,6 +27,15 @@ Update to latest selinux-policy:
   pkg.installed:
     - name: selinux-policy
 
+Install python36-ldap python package:
+  pip.installed:
+    - name: python36-ldap
+    - bin_env: /usr/bin/pip3
+    - target: /usr/lib64/python3.6/site-packages/
+    - require:
+      - Ensure cryptography python package absent
```
Comment (nitin-seagate) [inline srv/components/misc_pkgs/openldap/install.sls:36]: Why cryptography package to be absent?

**Review by nitin-seagate — COMMENTED** [review]
@sradhanand-pati  Please check with @ypise We just need to update provisionner_requirements.txt

**Comment by nitin-seagate** [comment]
Recreate new one after discussions

File: srv/components/misc_pkgs/openldap/install.sls (hunk 1 above)
Comment (sradhanand-pati) [reply srv/components/misc_pkgs/openldap/install.sls:36]: Oops... My bad.

**Pull request closed without merging**